src/charm.py

```diff
@@ -111,38 +111,48 @@ def _on_collect_unit_status(self, event: CollectStatusEvent):  # noqa C901
             # teardown code is necessary to perform if we're removing the
             # charm.
             event.add_status(BlockedStatus("Scaling is not implemented for this charm"))
+            logger.info("Scaling is not implemented for this charm")
             return
 
         for relation in ["database", "fiveg_nrf", "certificates"]:
             if not self._relation_created(relation):
                 event.add_status(BlockedStatus(f"Waiting for {relation} relation"))
+                logger.info(f"Waiting for {relation} relation")
```

Comment:
Our style guide prohibits f-strings in logging calls.

```suggestion
                logger.info("Waiting for %s relation", relation)
```

Reply:
Done